code/solr/tasks/Solr_Configure.php

```diff
@@ -9,7 +9,7 @@
 
 class Solr_Configure extends Solr_BuildTask
 {
-    private static $segment = 'Solr_Configure';
+    private static $segment = 'SilverStripeFullTextSearchSolrTasksSolr_Configure';
```

Comment:
We should keep the name of old task here (on Solr_Reindex as well). `$segment` is there after reading the migration guide to make the task backwards-compatible with existing code. It doesn't need to be this wordy  :)